diff -up gnome-video-arcade-0.8.0/src/Makefile.am.libX11 gnome-video-arcade-0.8.0/src/Makefile.am
--- gnome-video-arcade-0.8.0/src/Makefile.am.libX11	2011-05-15 20:00:58.700613506 -0400
+++ gnome-video-arcade-0.8.0/src/Makefile.am	2011-05-15 20:01:16.982427343 -0400
@@ -81,7 +81,7 @@ EXTRA_gnome_video_arcade_SOURCES = \
 gnome_video_arcade_LDADD = \
 	@GLIB_LIBS@ @GTK_LIBS@ @GIT_LIBS@ \
 	@GSTREAMER_LIBS@ @MAME_BACKEND@ \
-	@SQLITE_LIBS@ @WNCK_LIBS@
+	@SQLITE_LIBS@ @WNCK_LIBS@ -lX11
 
 gnome_video_arcade_DEPENDENCIES = \
 	@MAME_BACKEND@
diff -up gnome-video-arcade-0.8.0/src/Makefile.in.libX11 gnome-video-arcade-0.8.0/src/Makefile.in
--- gnome-video-arcade-0.8.0/src/Makefile.in.libX11	2011-05-15 20:00:53.240668700 -0400
+++ gnome-video-arcade-0.8.0/src/Makefile.in	2011-05-15 20:01:07.095527510 -0400
@@ -352,7 +352,7 @@ EXTRA_gnome_video_arcade_SOURCES = \
 gnome_video_arcade_LDADD = \
 	@GLIB_LIBS@ @GTK_LIBS@ @GIT_LIBS@ \
 	@GSTREAMER_LIBS@ @MAME_BACKEND@ \
-	@SQLITE_LIBS@ @WNCK_LIBS@
+	@SQLITE_LIBS@ @WNCK_LIBS@ -lX11
 
 gnome_video_arcade_DEPENDENCIES = \
 	@MAME_BACKEND@
